app/email_batching.go

```diff
@@ -107,7 +107,7 @@ func (job *EmailBatchingJob) CheckPendingEmails() {
 	// without actually sending emails
 	job.checkPendingNotifications(time.Now(), job.server.sendBatchedEmailNotification)
 
-	mlog.Debug(fmt.Sprintf("Email batching job ran. %v user(s) still have notifications pending.", len(job.pendingNotifications)))
+	mlog.Debug("Email batching job ran. User(s) still have notifications pending.", mlog.Int("number_of_users", len(job.pendingNotifications)))
```

Comment:
Since we're touching, maybe get rid of the awkward `User(s)` and just say, `Some users still have...`?

Reply:
Done